packages/e2e/test/wallet/SingleAddressWallet/delegation-rewards.test.ts

```diff
@@ -114,6 +115,7 @@ describe('delegation rewards', () => {
         const tx = await txBuilder.addOutput(txOut).build();
         assertTxIsValid(tx);
         await (await tx.sign()).submit();
+        await delay(1000);
```

Comment:
Which environment is it not working on without the delay? It was working on local network (including CI)

Reply:
removed from the PR